cannon/mipsevm/tests/evm_multithreaded_test.go

```diff
@@ -1179,10 +1179,14 @@ func TestEVM_UnsupportedSyscall(t *testing.T) {
 			// Setup basic getThreadId syscall instruction
 			state.Memory.SetMemory(state.GetPC(), syscallInsn)
 			state.GetRegistersRef()[2] = syscallNum
-
+			proofData := state.EncodeThreadProof()
+			insnProof := state.GetMemory().MerkleProof(state.GetPC())
+			proofData = append(proofData, insnProof[:]...)
```

Comment:
This can be simplified with your new helper:
```suggestion
			proofData := multiThreadedProofGenerator(t, state)
```